IM_calculation/IM/structural_response.py

```diff
@@ -0,0 +1,436 @@
+#!/usr/bin/env python
+
+import numpy as np
+
+
+def sa_sd_time(
+    acc, dt, t1_range=None, t1min=1e-06, t1max=5, nt=100, c=0.05, g=9.81, m=1
+):
+    # t1_range: give values instead of calculating them from min, max, nt
+    # t1min: minimum time period to be considered in the design acceleration spectra (seconds).
+    #        0 not chosen to avoid placing 0 in the denominator in later calculations.
+    # t1max: maximum time period to be considered in the design acceleration spectra (seconds).
+    # nt: number of points to be considered in the design acceleration spectra.
+    # c: damping  (c=0.05 represents 5% damping).
+    # g: 9.81 converts acceleration to ms-2;  (if g=1, unit will be in g because the unit of the acceleration record input is in g).
```

Comment:
I'm not sure what you mean here. Default behaviour is to convert to ms-2, but if a value of 1 is given then the units are in g

Reply:
It should change to be a true/false flag that changes it to g or not, rather than having to give conversion factors yourself?

Reply:
Yeah. Acceleration should defaultly be in g - but there are a few cases where we have acceleration in cms-2.

So a flag would be perfect rather than having to specify a conversion factor. 

We need to update the comment to say what units are taken in and subsequently what units are outputted.

Reply:
I think the comment may be incorrect as the code had the same outputs as Jaes work